README.md

```diff
@@ -59,6 +59,7 @@ If you need to remove Contiv from Docker Swarm and return to your original state
   ` <br> For example: <br> `./install/k8s/install.sh -n <netmaster DNS/IP> -a https://apic_host:443 -u apic_user -p apic_password -l topology/pod-xxx/node-xxx -d phys_domain -e not_specified -m no
   `
  <br> where `$CONTIV_MASTER` is the Contiv proxy or Net Master IP.
+* To install Contiv with a custom infra network and gateway: <br>`./install/k8s/install.sh -g <GATEWAY IP> -i <SUBNET>`
```

Comment:
@unclejack, can you add a complete example. Atleast a -n <netmaster DNS/IP> is a required parameter. If folks try to run it just as it is specified here, the installer will give out an error.